feat: Adding cssLoaderOptions to extend css-loader (#52)
spothero/ace: docs/settings-webpack.md

#### `plugins`
An array of [extra plugins](https://webpack.js.org/configuration/plugins) to add to the production build.

## `cssLoaderOptions`

> An object to pass to `css-loader` options, all supported options can be found [here](https://github.com/webpack-contrib/css-loader#options)

Reviewer: ```suggestion
An object of options to pass to [css-loader](https://github.com/webpack-contrib/css-loader#options).
```

Author: oh! we got ourselves a copy writer, updated
